includes/menu.inc

```diff
@@ -18,11 +18,6 @@ function oe_bootstrap_theme_preprocess_menu(&$variables, $hook) {
   $items = $variables['items'];
   foreach ($items as $key => $item) {
```

Comment:
Please, Carlos, check if this can be done:
foreach ($variables['items'] as $key => $item) {